test/unit_test/DistributionallyRobustUnitTest.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
Can this just be a python script instead of a notebook?

Also can this be run with the other tests?

Reply:
converted to script.  it just generates the test cases.  added comments to this effect.